genomic_data_service/region_indexer.py

```diff
@@ -274,9 +285,52 @@ def fetch_datasets(files, datasets):
         fetch_documents(dataset)
         datasets[dataset["accession"]] = dataset
 
+def get_encode_accessions(assemblies):
```

Comment:
I think this function would benefit from getting broken down into smaller, more concentrated, bits. We should write the pickle explicitly somewhere instead of a side-effect. I wonder if we should check somewhere whether the assemblies passed to the script are supported ones (maybe already at the argparse phase)?